openslides/assignments/migrations/0003_assignmentrelateduser_weight.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please use only one new migration file name give it a explicit name like `0003_candidate_weight.py`.